### ISSUE TITLE
Ability to trigger compiler on import

### ISSUE THREAD

**Issue by stefanv** [serge-sans-paille/pythran#2080]
In Cython, you can do:

```
import pyximport
pyximport.install()
```

Thereafter, if you pull in a module with a .pyx extension, it gets compiled in the background.

This may be a bit much for Pythran (since you can't use the extension to check what type of file it is), but perhaps something like:

```
import pythran
my_module = pythran.compile('my_module.py')
```

**Comment by serge-sans-paille** [comment]
Feature implemented in #2080, please let me know if that's close enough to what you were looking for. There's no caching mechanism etc, so... it's not for jit compilation :-)

**Comment by stefanv** [comment]
#2081
